include/ola/Constants.h

```diff
@@ -82,6 +82,39 @@ static const uint16_t OPEN_LIGHTING_ESTA_CODE = 0x7a70;
  */
 static const uint32_t MAX_UNIVERSE = 0xffffffff;
 
+/**
+ * @brief The minimum break time for a DMX frame (in microseconds).
+ */
+static const int DMX_BREAK_TIME_MIN = 1204;
+
+/**
+ * @brief The maximum break time for a DMX frame is 1 second (in microseconds).
+ */
+static const int DMX_BREAK_TIME_MAX = 1000000; 
```

Comment:
Remove the extra whitespace:
```suggestion
static const int DMX_BREAK_TIME_MAX = 1000000;
```